src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/2.sql

```diff
@@ -364,6 +364,274 @@ CREATE NONCLUSTERED INDEX IX_ChangeFeed_StudyInstanceUid_SeriesInstanceUid_SopIn
     SopInstanceUid
 )
 
+/*************************************************************
```

Comment:
I'm thinking if we should put all custom tag changes together, and put on the tail, mark as **version2**, so that can easily differentiate 2 vs 1.sql

Reply:
THis shoud be fine. 2.diff.sql will capture that once you add it.

Reply:
Hi Smitha, would 2.diff.sql be generated from any tool?

Reply:
No, needs to be manually created